packages/marshal/src/marshal.js

```diff
@@ -592,10 +596,21 @@ export function makeMarshal(
               // summary. If we do that, we could allocate some random
               // identifier and include it in the message, to help
               // with the correlation.
+
+              errorCount += 1;
+              // TODO in order to reliably correlate sent and received errors
+              // we will need something more globally identufying than the count
+              // relative to this marshal instance.
+              const errorId = `error#${errorCount}`;
+              assert.note(val, d`Sent as ${errorId}`);
+              // TODO we need to instead log to somewhere hidden
+              // to be revealed when correlating with the received error.
+              console.error('Temporary logging of sent error', val);
```

Comment:
If you make this a `console.log`, it will only display when `agoric start` is run with `-v`.

Reply:
What does `agoric start` do or not do depending on `-v`?

What about `yarn test`? Any similarly convenient way to make it less noisy?

Reply:
That's what anylogger gets you (wrapping the console).  None of the tests customize it, so they are noisy.

Reply:
Changed to `console.log`. Resolving.